components/segment-bridge/OWNERS

```diff
@@ -0,0 +1,5 @@
+# See the OWNERS docs: https://go.k8s.io/owners
+
+approvers:
+- Omeramsc
+- ifireball
```

Comment:
missing EOF newline

Same goes for multiple other files

Reply:
fixed